<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find finders_keepers)/params/without_map_params/move_base_params.yaml" command="load"/>

    <!-- local costmap parameters -->
    <rosparam file="$(find finders_keepers)/params/without_map_params/global_planner_params.yaml" command="load" />

    <!-- local costmap parameters -->
    <rosparam file="$(find finders_keepers)/params/without_map_params/local_planner_params.yaml" command="load" />

    <!-- observation sources located in common_costmap_params.yaml -->
    <rosparam file="$(find finders_keepers)/params/without_map_params/common_costmap_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find finders_keepers)/params/without_map_params/common_costmap_params.yaml" command="load" ns="local_costmap" />

    <!-- global costmap parameters (with laser) -->
    <rosparam file="$(find finders_keepers)/params/without_map_params/global_costmap_params.yaml" command="load" />

    <!-- local costmap parameters -->
    <rosparam file="$(find finders_keepers)/params/without_map_params/local_costmap_params.yaml" command="load" />

    <remap from="odom" to="odometry/filtered" />

  </node>

</launch>
